.github/workflows/precheck.yml

```diff
@@ -0,0 +1,26 @@
+name: "Trigger precheck"
+on:
+  issue_comment:
+    types:
+    - created
+
+jobs:
+  precheck:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: Install jq
+      run: sudo apt-get install jq
```

Comment:
Are we really installing a package for every comment posted to every PR?  This feels wasteful to me.

Reply:
Yes, it is wasteful. Equally, we clone the entire repository just to be able to access the script file (we could inline the script in the workflow file to avoid this). The whole process takes very little time, but still...